### ISSUE TITLE
Cannot build Android app for release because of compileSdk = 35

### ISSUE THREAD

**Issue by maelchiotti** [GregoryConrad/mimir#376]
Hi, I cannot build my app for release because flutter_mimir [requires SDK 35](https://github.com/GregoryConrad/mimir/blob/1c78e5c58ff593dd23fc35b3b9d93fc90263e24d/packages/flutter_mimir/android/build.gradle#L30).

According to [SO](https://stackoverflow.com/questions/78778617/android-resource-linking-failed-with-android-35-sdk-during-flutter-release-build), I could upgrade my gradle plugin. However Flutter [uses SDK 34](https://github.com/flutter/flutter/blame/370f40e6dfbf742d7b980dc7cc38152975ed8d84/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy) by default, so is there a reason you used 35? If not I think using the default Flutter version would be better:

```
compileSdk = flutter.compileSdkVersion
ndkVersion = flutter.ndkVersion
```

**Comment by GregoryConrad** [comment]
> by default, so is there a reason you used 35?

No, I just introduced that because I was updating my Android build.gradle boilerplate since it was too far out of date. The new boilerplate used 35.

> ```
> compileSdk = flutter.compileSdkVersion
> ndkVersion = flutter.ndkVersion
> ```

That looks like what I actually wanted, thanks! #377

**Comment by GregoryConrad** [comment]
Released a new version, 0.1.5, plz wait a few hours before attempting to use it (otherwise your pub cache will be messed up, since the binaries haven't been built by CI yet)

**Comment by maelchiotti** [comment]
Awesome! I will confirm it's fixed when I try again.
